collections.md

```diff
@@ -2417,9 +2417,9 @@ $ratio = [
     'eur' => 1.22,
 ];
 
-$collection->reduce(function (int $carry, int $value, int $key) use ($ratio) {
+$collection->reduce(function (int $carry, int $value, string $key) use ($ratio) {
     return $carry + ($value * $ratio[$key]);
-});
+}, 0);
```

Comment:
In a strict setting, this example would fail as it will attempt to pass `null` to an `int` typed parameter.